src/onboarding/registration/MockedStatSigFeatureFlag.ts

```diff
@@ -0,0 +1,6 @@
+export const shallShowAlternatePlaceholder = () => {
```

Comment:
```suggestion
export const shouldShowNamePlaceholderVariation = () => {
```
see comment below, same idea